.circleci/workflows.yml

```diff
@@ -448,6 +448,10 @@ commands:
                   If('<< parameters.source >>' -eq 'CodeArtifact'){dotnet nuget push $NupkgPath --source << parameters.source >>}
                   ElseIf('<< parameters.source >>' -eq 'PSGallery'){dotnet nuget push $NupkgPath --source << parameters.source >> --api-key $env:NuGetApiKey}
                   Else {Write-Error "Unknown source value: << parameters.source >>"}
+            - persist_to_workspace:
```

Comment:
I'm adding the nuspec to the files written back to git. The old nuspec file in our repository was several versions old